server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/LocalStorageManager.scala

```diff
@@ -324,9 +362,9 @@ private[worker] final class LocalStorageManager(
         }
```

Comment:
Should we change `LocalStorageManager` to `StorageManager`?
Or we will implement different `StorageManager` for each storage?

Reply:
Yes, you are right. I should change it to `StorageManager`.